src/state/upload/upload-reducers.js

```diff
@@ -37,9 +38,24 @@ export default function (state = initialState, action) {
 
         case uploadActionTypes.FETCH_ITEMS_SUCCEEDED: {
             const { groups } = action.data;
+            let ID = null;
```

Comment:
Ah, I missed that this was for the upload screen only -- thought it was also potentially affecting the sidebar.